client/src/components/NavbarBottom.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import styled from "styled-components/macro";
+import { NavLink } from "react-router-dom";
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 3rem;
+  width: 100vw;
+  background: var(--primary-first);
+  border: none;
+`;
+
+const StyledNavLink = styled(NavLink)`
+  fill: var(--neutral-mid-N60);
+  text-decoration: none;
+`;
+
+const activeNavLink = {
+  fill: `var(--primary-third)`,
+};
+
+const NavbarBottom = () => {
+  return (
+    <Container>
+      <StyledNavLink to="/" activeStyle={activeNavLink} exact={true}>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          enableBackground="new 0 0 24 24"
+          height="24"
+          viewBox="0 0 24 24"
+          width="24"
+        >
+          <g>
+            <rect fill="none" height="24" width="24" />
+          </g>
+          <g>
+            <g>
+              <path d="M19,9.3V4h-3v2.6L12,3L2,12h3v8h6v-6h2v6h6v-8h3L19,9.3z M17,18h-2v-6H9v6H7v-7.81l5-4.5l5,4.5V18z" />
+              <path d="M10,10h4c0-1.1-0.9-2-2-2S10,8.9,10,10z" />
+            </g>
+          </g>
+        </svg>
```

Comment:
You could import the SVGs as ReactComponent to keep the file cleaner

Reply:
Done ✅ 
See cd7e8e1